README.md

```diff
@@ -240,10 +240,37 @@ License: By requesting and using the Container OS Image for Windows containers,
 
 ### Telemetry
 
-By default, PowerShell collects the OS description and the version of PowerShell (equivalent to `$PSVersionTable.OS` and `$PSVersionTable.GitCommitId`) using [Application Insights](https://azure.microsoft.com/services/application-insights/).
-To opt-out of sending telemetry, create an environment variable called `POWERSHELL_TELEMETRY_OPTOUT` set to a value of `1` before starting PowerShell from the installed location.
+Sending telemetry is controlled via an environment variable called `POWERSHELL_TELEMETRY_OPTOUT`.
+Set the value to `1`, `yes`, or `true` before starting PowerShell to disable telemetry.
+Setting the value to `0`, `no`, `false`, or not creating the variable will enable telemetry.
```

Comment:
Isn't it more accurate that if it's anything other than `1`,`yes`, or `true` then it defaults to false meaning to send telemetry?